lib/ReactViewModels/MouseCoords.js

```diff
@@ -133,20 +134,18 @@ export default class MouseCoords {
 
         const terrainProvider = globe.terrainProvider;
         if (!(terrainProvider instanceof EllipsoidTerrainProvider)) {
-            clearTimeout(this.accurateHeightTimer);
-            this.accurateHeightTimer = setTimeout(()=> {
-                this.sampleAccurateHeight(terrainProvider, position);
-            }, this.accurateSamplingDebounceTime);
+            if(defined(this.accurateHeightTimer)) this.accurateHeightTimer.cancel();
```

Comment:
I think this is just using debounce as a timer, rather than actually using it to debounce.  All our existing code to do the debounce is still in place.  By really using `debounce`, you should be able to remove our `debounceSampleAccurateHeight` function entirely.